docs/sources/setup-grafana/installation/helm/index.md

```diff
@@ -0,0 +1,379 @@
+
+1. Save the decoded password to a file on your machine.
+
+1. To access Grafana service on the web browser, run the following command:
+
+   ```bash
+   export POD_NAME=$(kubectl get pods --namespace monitoring -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=my-grafana" -o jsonpath="{.items[0].metadata.name}")
+   ```
+
+   The above command will export a shell variable named `POD_NAME` that will save the complete name of the pod which got deployed.
+
+1. Run the following port forwarding command to direct the Grafana pod to listen to port `3000`:
+
+   ```bash
+   kubectl --namespace monitoring port-forward $POD_NAME 3000
+   ```
+
+   For more information about port-forwarding, refer to [Use Port Forwarding to Access Applications in a Cluster](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+1. Navigate to `127.0.0.1:3000` in your browser.
+
+1. The Grafana sign-in page appears.
+
+1. To sign in, enter `admin` for the username.
+
+1. For the password paste it which you have saved to a file after decoding it earlier.
+
+## Customize Grafana default configuraiton
+
+Helm is a popular package manager for Kubernetes. It bundles Kubernetes resource manifests to be re-used across different environments. These manifests are written in a templating language, allowing you to provide configuration values via `values.yaml` file, or in-line using Helm, to replace the placeholders in the manifest where these configurations should reside.
+
+The `values.yaml` file allows you to customize the chart's configuration by specifying values for various parameters such as image versions, resource limits, service configurations, etc.
+
+By modifying the values in the `values.yaml` file, you can tailor the deployment of a Helm chart to your specific requirements by using the helm install or upgrade commands. For more information about configuring Helm, refer to [Values Files](https://helm.sh/docs/chart_template_guide/values_files/).
+
+### Download the values.yaml file
+
+In order to make any configuration changes, download the `values.yaml` file from the Grafana Helm Charts repository:
+
+https://github.com/grafana/helm-charts/edit/main/charts/grafana/values.yaml
+
+{{% admonition type="note" %}}
+Depending on your use case requirements, you can use a single YAML file that contains your configuration changes or you can create multiple YAML files.
+{{% /admonition %}}
+
+### Enable persistent storage **(recommended)**
+
+By default, persistent storage is disabled, which means that Grafana uses ephemeral storage, and all data will be stored within the container's file system. This data will be lost if the container is stopped, restarted, or if the container crashes.
+
+It is highly recommended that you enable persistent storage in Grafana Helm charts if you want to ensure that your data persists and is not lost in case of container restarts or failures.
```

Comment:
```suggestion
It is recommended that you enable persistent storage in Grafana Helm charts if you want to ensure that your data persists and is not lost in case of container restarts or failures.
```